### ISSUE TITLE
Receiving "org.schemaspy.output.diagram.DiagramException: Dot missing or invalid version" error with GraphViz 5.0.1 and 6.0.1

### ISSUE THREAD

**Issue by kburchfiel** [schemaspy/schemaspy#963]
## Expected Behavior

I am expecting to see diagrams of my database when running Schemaspy.

## Current Behavior

No diagrams are created; instead, I receive the following error message:

" org.schemaspy.output.diagram.DiagramException: Dot missing or invalid version
        at org.schemaspy.output.diagram.graphviz.GraphvizDot.generateDiagram(GraphvizDot.java:193)
        at org.schemaspy.output.diagram.DiagramFactory.generateOrphanDiagram(DiagramFactory.java:58)"

## Possible Solution

Should I be using an earlier version of GraphViz?

## Context

* Database type (e.g. MySQL): PostgreSQL
* Database driver version (is you use a special one):
* Your DDL (sample how to reproduce the issue):
* Full command including arguments how you run SchemaSpy:
* java -jar C:\Users\kburc\Downloads\schemaspy-6.1.0.jar -configFile path\to\file\schemaspy_postgresql_creds.properties

I also linked to GraphViz in my .properties file using the following line:
schemaspy.gv=C:\Program Files\Graphviz\bin

## Your Environment

* Version used: 6.1.0 (Running from my Downloads folder)
* Environment name and version (e.g. Chrome 39): Windows command prompt
* Operating System and version (e.g. Windows 10): Windows 10
* Java version: OpenJDK 19

**Comment by douglassix-dkt** [comment]
Hi,
This issue seems fixed in the 6.1.1-SNAPSHOT version.
After a clone of this repo and a `mvn package -DskipTests`, the new jar is generated in the `./target` folder. I used it with the command line 
```
 java -jar ./schemaspy-6.1.1-SNAPSHOT.jar -t pgsql -dp ./postgresql-42.4.1.jar -db spiddata -host localhost -port 5432 -s spiddata -u spiddata -p spiddata -o ./schemaspy_doc -imageformat svg -renderer :cairo 
```
And got my documentation generated.

**Comment by npetzall** [comment]
If you don't want to clone and build, there's a download link in the Readme for the snapshot build.

**Comment by marcelstoer** [comment]
Should be closed as 6.2.0 is out now.

**Comment by npetzall** [comment]
Verified ok in 6.1.1-SNAPSHOT and release in 6.2.0.
Re-open if needed
